torch/csrc/jit/import.cpp

```diff
@@ -467,8 +467,13 @@ void ScriptModuleDeserializer::loadParams(
   std::shared_ptr<script::Module> module = moduleLookup_(moduleStack_);
   for (int i = 0; i < module_def.parameters_size(); ++i) {
     const torch::ParameterDef& param_def = module_def.parameters(i);
-    uint64_t tensor_id = caffe2::stoull(param_def.tensor_id());
-    at::Tensor tensor = loadTensor(tensor_id);
+    at::Tensor tensor = at::Tensor();
+    // If param_def does not have tensor_id field, it indicates an undefined
+    // tensor, otherwise we get the tensor_id and load the tensor
+    if (param_def.has_tensor_id()) {
```

Comment:
I'm not sure that 'has' propagates properly through json serialization and it's also non proto3 compatible. Better to check explicit value whether it's an empty string  or not

Reply:
Aha, I thought the field set/unset in proto is checked by the `has` call across proto2/3, I will double check proto3 on this.